packages/entity/src/EntityFields.ts

```diff
@@ -1,7 +1,9 @@
-import { validate as validateUUID } from 'uuid';
```

Comment:
Should we make this a dev dependency?

Reply:
It turns out `uuid` is used by `StubDatabaseAdapter`, which ends up getting exported as part of the package's production interface. So for now the right thing to do AFAIK is to keep it as as dependency. If we introduced `@expo/entity-testing` as a separate package, we could make uuid a devDep of `@expo/entity` but I don't think that'd get us much.